.gitignore

```diff
@@ -25,3 +25,9 @@ yarn-error.log*
 .idea
 
 .env
+
+
+# TV 
+public/charting_library
```

Comment:
Given that these files are required to run the UI now, can we add

1) How to retrieve these files from trading view
2) Where to place these fields in the repo

To the README.

Reply:
I just learned the README already has these instructions. Removed the steps that this PR implements.